app/views/manifests/show.json.rabl

```diff
@@ -0,0 +1,9 @@
+object false
+
+node(:name)        { Setting.site_title }
+node(:description) { strip_tags(Setting.site_description.presence || I18n.t('about.about_mastodon')) }
+node(:icons)       { [{ src: '/android-chrome-192x192.png', sizes: '192x192', type: 'image/png' }] }
+node(:theme_color) { '#282c37' }
```

Comment:
A `background_color` would be nice as well, since it's used as the background in the splash screen and thus helps improve "perceived performance." It seems for us the best value would be `$ui-secondary-color`.